src/main/java/unicash/logic/commands/SummaryCommand.java

```diff
@@ -12,11 +12,15 @@ public class SummaryCommand extends Command {
 
     public static final String MESSAGE_USAGE = CommandType.SUMMARY.getMessageUsage();
 
-    public static final String SHOWING_SUMMARY_MESSAGE = "Opened UniCa$h summary window.";
+    public static final String MESSAGE_SHOWING_SUMMARY_SUCCESS = CommandType.SUMMARY.getMessageSuccess();
+
+    public static final String MESSAGE_NOT_SHOWING_SUMMARY = "You have no expenses to summarize.";
 
     @Override
     public CommandResult execute(Model model) {
-        model.updateExpenseSummary();
-        return new CommandResult(SHOWING_SUMMARY_MESSAGE, false, false, true);
+        if (model.hasExpenses()) {
+            return new CommandResult(MESSAGE_SHOWING_SUMMARY_SUCCESS, false, false, true);
+        }
+        return new CommandResult(MESSAGE_NOT_SHOWING_SUMMARY, false, false, false);
```

Comment:
I also realise that you could use `new CommandResult(MESSAGE_NOT_SHOWING_SUMMARY)` alone right? Since that constructor sets all the booleans to `false` alreaedy